package.json

```diff
@@ -35,8 +35,8 @@
     "react": "15.4.1",
     "react-dom": "15.4.1",
     "react-redux": "5.0.1",
-    "react-router": "3.0.0",
-    "react-router-redux": "4.0.7",
+    "react-router-dom": "4.0.0",
+    "react-router-redux": "5.0.0-alpha.3",
```

Comment:
This feels a bit rough to be using an alpha package.  Many of us are using this library for projects and these things will inevitably break into beta+